scout/server/blueprints/variants/templates/variants/variants.html

```diff
@@ -55,8 +55,9 @@
          <thead class="thead table-light">
             <tr>
               <th style="width:2%"></th>
-              <th style="width:12%" title="Rank position">Rank</th>
+              <th style="width:5%" title="Rank position">Rank</th>
               <th style="width:5%" title="Rank score">Score</th>
+              <th style="width:7%">Callers</th>
```

Comment:
We chase these % so hard: I don't think this is useful enough here. More clinvar detail, quality measures, more in silico predictor results, more gene details all would go before in mind: you would not dismiss or be interested in a variant based on the RD SNV caller these days.

Reply:
That's fine, we can remove that callers colum completely